$OpenBSD: patch-ui_widget_widget_c,v 1.1 2021/02/19 19:02:09 fcambus Exp $

Remove duplicate definition of widget_options_settings, this fixes the
build with -fno-common.

Upstream commit b73d549dac901923051047ddbe982b8be0e47ccf.

Index: ui/widget/widget.c
--- ui/widget/widget.c.orig
+++ ui/widget/widget.c
@@ -90,9 +90,6 @@ typedef struct widget_recurse_t {
 
 static widget_recurse_t widget_return[10]; /* The stack to recurse on */
 
-/* The settings used whilst playing with an options dialog box */
-settings_info widget_options_settings;
-
 static int widget_read_font( const char *filename )
 {
   utils_file file;
